### ISSUE TITLE
Postman 8.0.3, not loading all history

### ISSUE THREAD

**Issue by remcolam** [a85/POSTMan-Chrome-Extension#9513]
Postman is not loading in the history, it is only loading a few days, and only once you scroll to the end, will it continue loading the next few days. 

When you search, it will not find any requests that have not yet been loaded. So trying to get to a request from a month ago, takes you several minutes of scrolling and then finally you can start searching for it. 

However, as soon as you restart postman, the history is empty again and the whole thing needs to be redone. 

**Environment Information**

 - Platform Type: Native desktop app & Postman on the web
 - Postman Version: 8.0.3
 - Operating System: Windows 10

**Comment by DannyDainton** [comment]
Hey @remcolam

Could you provide a short recording of this issue please?

**Comment by remcolam** [comment]
Here it is, very short, in a small screen, to stay under the 10 mb...

https://user-images.githubusercontent.com/11367055/106435109-bb846f00-6472-11eb-8a50-a5f6ce7ddb82.mp4

**Comment by DannyDainton** [comment]
While this is loading, is the sync status still 'syncing' - You should be able to see this if you click/hover over the sync icon.

**Comment by remcolam** [comment]
No, it is green.

![image](https://user-images.githubusercontent.com/11367055/106435888-b8d64980-6473-11eb-8d48-cb43c539f93b.png)

and the tooltip says:

![image](https://user-images.githubusercontent.com/11367055/106435925-c4297500-6473-11eb-8235-7a6abfa48dae.png)

**Comment by mccannt** [comment]
Closing out this issue as its a duplicate of [9566](https://github.com/postmanlabs/postman-app-support/issues/9566)

**Comment by remcolam** [comment]
Tss, this is not a duplicate, the other one is a duplicate, this one was earlier ;)

**Comment by Naif95** [comment]
I have the same issue and it's very annoying for me as a developer working on many different projects.

**Comment by UjjualUSspericorn** [comment]
Have they fixed this issue ?

**Comment by Naif95** [comment]
No

**Comment by andersonb** [comment]
it hasn't been fixed yet
